apps/mobile/src/components/InsideRouter/atoms.ts

```diff
@@ -18,6 +25,20 @@ export const refreshBtcPriceActionAtom = atom<undefined, [], Task<boolean>>(
   (get, set) => {
     const api = get(privateApiAtom)
     const {t} = get(translationAtom)
+    const btcPriceLastUpdateAt = get(btcPriceLastUpdateAtAtom)
+    const timeDifferenceSinceLastUpdate = Math.round(
```

Comment:
Can you wrap this in `if(btcpriceLastUpdateAt)`? (or use `UnixMilliseconds` and make it non nullable with default 0 so it always runs with initial value).